src/span.js

```diff
@@ -222,14 +168,11 @@ export default class Span {
     }
 
     /**
-     * Indicates that the unit of work represented by the span is complete or
-     * has otherwise been terminated.
-     *
-     * All Span objects must have finish() called on them before they are
-     * reported to the backend implementation.
+     * Sets the end timestamp and finalizes Span state.
      *
-     * Once `finish()` is called on a Span object, the behavior of all methods
-     * on the object is considered undefined.
+     * With the exception of calls to Span.context() (which are always allowed),
```

Comment:
This is a bit of an aside: that *any* call is still allowed after `finish()` essentially prevents any sort of object pooling/reuse optimizations that could be done if nothing were valid to call after `finish()`.  I haven't followed / been following the spec discussion closely: is it *necessary* that `context()` is callable after `finish()` (or nice-to-have)?